metricbeat/module/windows/perfmon/perfmon_test.go

```diff
@@ -15,7 +15,8 @@
 // specific language governing permissions and limitations
 // under the License.
 
-// +build integration windows
+// +build integration
+// +build windows
```

Comment:
Without this change this integration test was being run on Linux. Multiple build tags on the same line are combined with a logical OR whereas multiple build tags on different lines are combined with an "AND". We want the latter here.